src/WorldStorage/NamespaceSerializer.h

```diff
@@ -23,4 +23,7 @@ namespace NamespaceSerializer
 	eMonsterType ToMonsterType(std::string_view a_ID);
 
 	std::pair<Namespace, std::string_view> SplitNamespacedID(std::string_view ID);
+
+	// Examples: Input: "wolf" -> Output: "Wolf", Input: "iron_golem" -> Output: "Iron Golem"
+	AString Prettify(AString a_Name, const bool a_IsTamed = false);
```

Comment:
This name needs to be more descriptive. Maybe `PrettifyEntityName`?

Reply:
This can be useable for any type of enum represented in the file.

this was at least my thought

Reply:
with an IsTamed parameter? No thanks! Rename to PrettifyEntityName, with IsTamed, and move the core logic to a new Prettify function (also exported, without IsTamed) would be a much better solution.

Reply:
Jup 👍